zeebe/protocol/src/main/resources/protocol.xml

```diff
@@ -58,6 +58,7 @@
       <validValue name="USER">41</validValue>
       <validValue name="CLOCK">42</validValue>
       <validValue name="AUTHORIZATION">43</validValue>
+      <validValue name="ROLE">44</validValue>
```

Comment:
💭 This will conflict with another PR. Guess we'll see which merges first 😄

Reply:
Yes, saw that